content/SCALE/SCALETutorials/Credentials/DirectoryServices/_index.md

```diff
@@ -9,266 +9,20 @@ aliases:
-* `AD.DOMAIN.COM` is the trusted domain
-* `user` is the user account name to authenticate.
-{{< /expand >}}
-
-{{< expand "Clean Up Active Directory" "v" >}}
-TrueNAS SCALE requires users to cleanly leave an Active Directory using the **Leave Domain** button under **Advanced Settings** to remove the AD object. 
-
-If the AD server moves or shuts down without you using **Leave Domain**, TrueNAS won't remove the AD object, and you will have to clean up the Active Directory.
-
-Go to **Credentials > Directory Services** and click **Show** next to **Advanced Settings**
-
-1. Clean out Kerberos settings by clicking **Settings** in the **Kerberos Settings** window and clearing the **Appdefaults Auxiliary Parameters** and **Libdefaults Auxiliary Parameters** boxes. You may also need to clear out leftover Kerberos Realms and Keytabs by clicking the <i class="material-icons" aria-hidden="true">delete</i> next to the remaining entries.
-2. Click the *Idmap* *Active Directory - Primary Domain* entry and clear out the Active Directory settings, then click **CONTINUE** to clear the Idmap cache.
-3. Go to **Network** and click *Settings* in the *Global Configuration* window. Remove the Active Directory Nameserver and enter a new one.
-4. Ensure all other network settings are correct.
-5. Go to **System Settings > Services** and change the workgroup to "WORKGROUP".
-6. Go to **Credentials> Directory Services** and edit the Active Directory config to the new domain.
-7. Make sure the Kerberos settings and Idmap are correct and SMB is running.
-{{< /expand >}}
-
-## Configuring LDAP In TrueNAS
-
-TrueNAS has an [Open LDAP](https://www.openldap.org/) client for accessing the information on an LDAP server. An LDAP server provides directory services for finding network resources like users and their associated permissions.
-
-{{< expand "Does LDAP work with SMB?" "v" >}}
-LDAP authentication for SMB shares is disabled unless you have configured and populated the LDAP directory with Samba attributes.
-The most popular script for performing this task is `smbldap-tools`.
-The LDAP server must support SSL/TLS, and you must import the certificate for the LDAP server CA.
-TrueNAS does not support non-CA certificates.
-{{< /expand >}}
-
-Go to **Credentials > Directory Services** and click **Configure LDAP**.
-
-![LDAPSCALE](/images/SCALE/LDAPSCALE.png "LDAP Options")
-
-Enter your LDAP server hostname, then enter your LDAP server Base and Bind domain names and the bind password. Check the **Enable** box to activate the server, then click **Save**.
-
-To further modify the LDAP configuration, click **Advanced Options**. See the [LDAP UI Reference article]({{< relref "/content/SCALE/SCALEUIReference/Credentials/DirectoryServices/LDAP.md" >}}) for details about advanced settings.
-
-![LDAPAdvancedSCALE](/images/SCALE/LDAPAdvancedSCALE.png "LDAP Advanced Options")
+To view Idmap and Kerberos Services, click **Show** next to **Advanced Settings**.
```

Comment:
Should this information be part of the _index page?

Reply:
Yes, I decided to leave that in the index page as it applied to both primary directory services. I've done this in a few articles in other sections.